docs/docs/Deployment/deployment-kubernetes.md

```diff
@@ -4,31 +4,27 @@ slug: /deployment-kubernetes
 ---
```

Comment:
I'm unable to comment on line 2 so I'm commenting here instead. The title "Kubernetes" does not have good SEO value. Write descriptive titles.